bears/vcs/git/GitCommitBear.py

```diff
@@ -204,3 +213,17 @@ def check_body(self, body,
             yield Result(self, 'Body of HEAD commit contains too long lines. '
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
+
+    def check_duplicate_shortlogs(self, shortlogs: list):
+        shortlogs_set = set(shortlogs)
+        if len(shortlogs_set) == len(shortlogs):
+            return False
+        else:
+            for log in shortlogs_set:
+                if log in shortlogs:
```

Comment:
`if log in shortlogs:`
why is this check necessary ?
A set can not have such elements which are not present in the parent list from which it is formed.